catalogs/catalog-jdbc-mysql/src/main/java/com/datastrato/gravitino/catalog/mysql/operation/MysqlDatabaseOperations.java

```diff
@@ -92,37 +87,19 @@ public String generateDropDatabaseSql(String databaseName, boolean cascade) {
 
   @Override
   public JdbcSchema load(String databaseName) throws NoSuchSchemaException {
-    try (final Connection connection = this.dataSource.getConnection()) {
-      String query = "SELECT * FROM information_schema.SCHEMATA WHERE SCHEMA_NAME = ?";
-      try (PreparedStatement preparedStatement = connection.prepareStatement(query)) {
-        preparedStatement.setString(1, databaseName);
-
-        // Execute the query
-        try (ResultSet resultSet = preparedStatement.executeQuery()) {
-          if (!resultSet.next()) {
-            throw new NoSuchSchemaException(
-                "Database %s could not be found in information_schema.SCHEMATA", databaseName);
-          }
-          String schemaName = resultSet.getString("SCHEMA_NAME");
-          // Mysql currently only supports these two attributes
-          String characterSetName = resultSet.getString("DEFAULT_CHARACTER_SET_NAME");
-          String collationName = resultSet.getString("DEFAULT_COLLATION_NAME");
-          Map<String, String> properties = new HashMap<>();
-          properties.put("CHARACTER SET", characterSetName);
-          properties.put("COLLATE", collationName);
-
-          JdbcSchema.Builder builder =
-              JdbcSchema.builder()
-                  .withName(schemaName)
-                  .withProperties(properties)
-                  .withAuditInfo(AuditInfo.EMPTY);
+    List<String> allDatabases = listDatabases();
```

Comment:
why not use getSchema in jdbc?

Reply:
There is not `getSchema` in `JdbcDatabaseOperations`, so what do you mean of `getSchema`?

Reply:
@FANNG1 Can you take a look at this?

Reply:
For MySQL and PG, you use different ways to get schema from JDBC, could you unify it? If not, what's the reason?

```java
  private ResultSet getSchema(Connection connection, String schemaName) throws SQLException {
    final DatabaseMetaData metaData = connection.getMetaData();
    return metaData.getSchemas(database, schemaName);
  }
```

Reply:
For PG, we can obtain the schema by name exactly . 
```
metaData.getSchemas(database, schemaName);
```

However,  we can only list all schemas through `metaData.getSchemas` or `metaData.getSchema`(I don't know the root reason, but that's the truth) and there exists a `listSchema` there, so I reuse it.

Reply:
ok, let's go ahead